scripts/clean/db.sh

```diff
@@ -1,9 +1,15 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
 # Cleans all data according to docs to prepare for wallets running:
 #     https://cardanodocs.com/technical/wallets/
 
 # TODO: support purescript API rebuilding?
 
-rm -rf ./run/*
+echo "Cleaning Cardano SL db..."
+
+rm -rf run/*
 rm -rf wallet-db/
 rm -rf db-abc/
```

Comment:
Yes, we can. This name `abc` is really old (and you're right, it's strange name). So what name can we use here?